std/numeric.d

```diff
@@ -2572,15 +2572,16 @@ GapWeightedSimilarityIncremental!(R, F) gapWeightedSimilarityIncremental(R, F)
 
 /**
 Computes the greatest common divisor of $(D a) and $(D b) by using
-Euclid's algorithm.
+an efficient algorithm such as $(HTTP https://en.wikipedia.org/wiki/Euclidean_algorithm, Euclid's)
+or $(HTTP https://en.wikipedia.org/wiki/Binary_GCD_algorithm, Stein's) algorithm.
```

Comment:
fixup in -> https://github.com/dlang/phobos/pull/4951

@andralex could you try to let people do such simple fixes directly on their PR?
Since two months maintainers can even edit their PR branch directly yourself.
I know that monitoring PRs is quite time consuming, but a PR should be at least somewhat self-consistent - at least s.t. it doesn't require immediate fixes (except for the changelog).
I think once you approve a PR other maintainers are happy to help to look over the final steps.